pkgs/tools/misc/undocker/default.nix

```
@@ -0,0 +1,25 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromSourcehut,
+}:
```

somebody is probably going to complain about this not following nixpkgs-fmt standards
```suggestion
{ lib
, buildGoModule
, fetchFromSourcehut
}:
```